man.Rmd

```diff
@@ -106,31 +145,35 @@ To summarize, there are four steps in the basic roxygen2 workflow:
 
 4.  Rinse and repeat until the documentation looks the way you want.
 
-### roxygen2 comments, blocks, and tags {#roxygen-comments}
+### roxygen2 comments, blocks, and tags {#sec-man-roxygen-comments}
 
 Now that you understand the basic workflow, we'll go into more detail about the syntax.
 roxygen2 comments start with `#'` and all the roxygen2 comments preceding a function are collectively called a **block**.
-Blocks are broken up by **tags**, which look like `@tagName tagValue`, and the content of a tag extends from the end of the tag name to the start of the next tag[^man-1].
-A block can contain text before the first tag which is called the **introduction**. By default, each block generates a single documentation **topic**, i.e. a single `.Rd` file[^man-2] in the `man/` directory
+Blocks are broken up by **tags**, which look like `@tagName tagValue`, and the content of a tag extends from the end of the tag name to the start of the next tag[^man-6].
+A block can contain text before the first tag which is called the **introduction**. By default, each block generates a single documentation **topic**, i.e. a single `.Rd` file[^man-7] in the `man/` directory
 .
 
-[^man-1]: Or the end of the block, if it's the last tag.
+[^man-6]: Or the end of the block, if it's the last tag.
 
-[^man-2]: The name of the file is automatically derived from the object you're documenting.
+[^man-7]: The name of the file is automatically derived from the object you're documenting.
 
 Throughout this chapter we'll show you roxygen2 comments from real tidyverse packages, focusing on stringr, since the functions there tend to be fairly straightforward, leading to documentation that's understandable with relatively little context.
-Here's a simple first example: the documentation for `stringr::str_unique()`.
+Here's a simple first example: the documentation for `stringr::str_unique()`[^man-8].
+
+[^man-8]: This may not perfectly reflect the current documentation for `str_unique()`.
+    Among other reasons, we're suppressing the use of roxygen features that are introduced later in this chapter.
```

Comment:
This function actually uses `@inheritParams` but I pretend like it doesn't at this point.

Reply:
I don't think folks are going to be comparing this to the literal files, but it doesn't hurt to be safe

Reply:
Yeah ok we can kill this footnote. We have a lot already in this chapter.